Utilities/Debugger/ITK.natvis

```diff
@@ -162,6 +162,6 @@
 
   <!-- This file is incomplete.
   You can extend it by adding custom visualizations for more ITK types.
-  If you do, please contribute: https://itk.org/Wiki/ITK/Git/Develop -->
+  If you do, please contribute: https://insightsoftwareconsortium.github.io/ITKWikiArchive/Wiki/ITK/Git/Develop -->
```

Comment:
This link is now broken. It used to look like this:
<img width="934" alt="Screenshot 2025-04-16 16 22 47" src="https://github.com/user-attachments/assets/cba27ca7-7919-4742-a0b1-09d5cf0027c3" />

Reply:
Updated.